### PR TITLE
Updating Create VM documentation for availabilitySet property.

### PR DESCRIPTION
**Pull request by avinashakka**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json [hunk 1]
```diff
@@ -7897,7 +7897,7 @@
...
+          "description": "Specifies information about the availability set that the virtual machine should be assigned to. Virtual machines specified in the same availability set are allocated to different nodes to maximize availability. For more information about availability sets, see [Manage the availability of virtual machines](https://docs.microsoft.com/azure/virtual-machines/virtual-machines-windows-manage-availability?toc=%2fazure%2fvirtual-machines%2fwindows%2ftoc.json). <br><br> For more information on Azure planned maintenance, see [Planned maintenance for virtual machines in Azure](https://docs.microsoft.com/azure/virtual-machines/virtual-machines-windows-planned-maintenance?toc=%2fazure%2fvirtual-machines%2fwindows%2ftoc.json) <br><br> Currently, a VM can only be added to availability set at creation time. The availability set to which the VM is being added to should also be under the same resource group. An existing VM cannot be added to an availability set. <br><br>This property cannot exist along with a non-null properties.virtualMachineScaleSet reference."
```
Comment (smotwani) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: The availability set to which the VM is being added to should be under the same resource group as the availability set resource.

Comment (avinashakka) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: Done.

**Comment by akning-ms** [comment]
Hi @hyonholee, @avinashakka request your review for this PR, can you help?

**Review by hyonholee — APPROVED** [review]

**Review by smotwani — APPROVED** [review]

**Comment by avinashakka** [comment]
> Hi @hyonholee, @avinashakka request your review for this PR, can you help?

@akning-ms Can you please merge the PR as I have the necessary approvals.

**Comment by avinashakka** [comment]
@akning-ms Can you please merge this PR. We need to get Azure documentation updated and that needs to consume all changes for ApiVersion: 2019-07-01 including changes from this PR.

**Pull request merged**